accelerometer/summariseEpoch.py

```diff
@@ -79,8 +80,10 @@ def getActivitySummary(epochFile, nonWearFile, summary,
         e = epochFile
     else:
         # Use python PANDAS framework to read in and store epochs
-        e = pd.read_csv(epochFile, parse_dates=['time'], index_col=['time'],
-            compression='gzip').sort_index()
+        e = pd.read_csv(
+            epochFile, index_col=['time'],
+            parse_dates=['time'], date_parser=accUtils.date_parser,
+        )
```

Comment:
We need special `date_parser` to parse through the time format that now contains timezones.